### ISSUE TITLE
Tour icons

### ISSUE THREAD

**Issue by hyanwong** [OneZoom/OZtree#807]
I think it would be really useful to have an (optional) "tour icon" for each tour, so that they can be easily distinguished. We can put the icon on the top right of the menubar.

I'm quite keen to store the icon as an SVG file (https://www.phylopic.org provides lots of good "silhouette" type ones that are public domain and could be modified, e.g. below for the frogs tour)

If we do allow an SVG icon, we could, I suppose, store the SVG in the database, which I would find neater than having to upload an image.

![frog](https://github.com/OneZoom/OZtree/assets/4699014/6622ac54-3fcb-4f98-968e-c3c8e1cd1351)

**Comment by jrosindell** [comment]
I like this idea a lot but don't think it's part of the "paradoxical frog" release.

**Comment by hyanwong** [comment]
> I like this idea a lot but don't think it's part of the "paradoxical frog" release.

I agree. However, if we do all think it is a good idea (and we can store the icons in the DB) then maybe we could make a DB field for them in the tours database, alongside the image_url one.
